src/SLCore/State/ActiveConfigScopeTracker.cs

```diff
@@ -131,7 +131,10 @@ public void RemoveCurrentConfigScope()
 
         using (asyncLock.Acquire())
         {
-            Debug.Assert(currentConfigScope != null);
+            if (currentConfigScope is null)
```

Comment:
I can't do anything about remove being called multiple times, so it's best to just ignore it